vm-aws-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,124 @@
+name: vm-aws-yaml
+description: A YAML program to create a virtual machine on AWS
+runtime: yaml
+
+template: 
+    description: A YAML program to create a virtual machine on AWS
+    config:
+        aws:region:
+            description: The AWS region to deploy into
+            default: us-west-2
+        instanceType:
+            description: Instance type of EC2
+            default: t2.micro
+        vpcNetworkCidr:
+            description: Network CIDR to use for new VPC
+            default: 10.0.0.0/16
+
+configuration:
```

Comment:
It's a remnant of the YAML launch, but basically everything from here down needs to be repeated in a new file called `Pulumi.yaml.append` (and needs to match what's in this file of course). You'll see this in other YAML templates as well if you want some examples. We should be able to remove these soon, though: https://github.com/pulumi/templates/issues/325